src/shared/ui/list/list.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { render, screen } from '@testing-library/react';
+import { List, cx } from './list';
+
+const data = ['listItem 1', 'listItem 2', 'listItem 3'];
+
+describe('List Component', () => {
+  it('renders list correctly', () => {
+    render(<List data={data} />);
+
+    const list = screen.getByTestId('list');
+
+    expect(list).toBeVisible();
```

Comment:
It might be better to check that the number of elements received is equal to the number of rendered ones.

`const listItems = screen.getAllByRole('list-item');
  expect(listItems).toHaveLength(3);`